prowler/providers/kubernetes/services/kubelet/kubelet_service.py

```diff
@@ -0,0 +1,29 @@
+import yaml
+
+from prowler.lib.logger import logger
+from prowler.providers.kubernetes.lib.service.service import KubernetesService
+from prowler.providers.kubernetes.services.core.core_client import core_client
+
+
+################## Kubelet ##################
+class Kubelet(KubernetesService):
+    def __init__(self, audit_info):
+        super().__init__(audit_info)
+        self.client = core_client
+
+        self.kubelet_config_maps = self.__get_kubelet_config_maps__()
+
+    def __get_kubelet_config_maps__(self):
+        try:
+            kubelet_config_maps = []
```

Comment:
Why are you using a list here? Please, always use a map for this objects, better to access to them if needed.

Reply:
It is a list with the dictionaries (config maps) that apply to this service.